src/Swarm/TUI/View.hs

```diff
@@ -785,6 +787,7 @@ drawKeyMenu s =
       , Just (NoHighlight, "^p", if isPaused then "unpause" else "pause")
       , Just (NoHighlight, "^o", "step")
       , Just (NoHighlight, "^zx", "speed")
+      , Just (if s ^. uiState . uiShowRobots then NoHighlight else Alert, "M-h", "hide robots")
```

Comment:
I am not sure if it is important enough to show in the UI (it is also in help) but it was useful while debugging. 😁

Reply:
I think it's fine.  We don't show some "obvious" keys in the UI (like keys for doing stuff at the REPL) but this one is definitely not obvious.